javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    OrderPlaced: 'August 4, 2020',
+    Total: '$34.00',
+    ShipTo: 'JS Masher',
+    items: [
+      {OrderNumber: '114-3941689-8772232'},
```

Comment:
very close! I would use a single object for each item, so all of the properties for each item are in a single object rather than multiple objects